influxdb/client.py

```diff
@@ -101,6 +107,49 @@ def __init__(self,
             'Content-type': 'application/json',
             'Accept': 'text/plain'}
 
+    @staticmethod
+    def from_DSN(dsn):
```

Comment:
I would make it ``def from_DSN(dsn, **kwargs):`` and pass all kwargs to the ``InfluxDBClient`` init.

That way people can configure things like timeout and verify_ssl.